src/encode/pattern/mod.rs

```diff
@@ -619,29 +649,37 @@ impl FormattedChunk {
 /// An `Encode`r configured via a format string.
 #[derive(Derivative)]
 #[derivative(Debug)]
-#[derive(Clone, Eq, PartialEq, Hash)]
+#[derive(Clone, Eq, PartialEq)]
 pub struct PatternEncoder {
     #[derivative(Debug = "ignore")]
     chunks: Vec<Chunk>,
     pattern: String,
+    color_map: HashMap<Level, Option<Color>>,
 }
 
 /// Returns a `PatternEncoder` using the default pattern of `{d} {l} {t} - {m}{n}`.
 impl Default for PatternEncoder {
     fn default() -> PatternEncoder {
-        PatternEncoder::new("{d} {l} {t} - {m}{n}")
+        PatternEncoder::new_with_colormap(DEFAULT_PATTERN_ENCODER, default_color_map())
```

Comment:
I need to think about this, by changing the default behavior I think this is a major semver change.

Reply:
This doesn't change the behavior, so what would be the conflict w/ semver? Also i believe this can be reverted to just call `PatternEncoder::new("{d} {l} {t} - {m}{n}")` since `PaternEncoder::new()` also uses the `default_color_map()` to construct the PaternEncoder.

Reply:
OH, you're 100% right sorry about that